user-images/staginghub/Dockerfile

```diff
@@ -1,8 +1,24 @@
-FROM earthlab/earth-analytics-python-env:41ae80f
```

Comment:
@kcranston i had a thought on this. you can actually setup docker to update an image when another image updates. if we used `latest` here that might ensure that when we update the ea envt and merge to master, it pushes updates over to the earthlabhubops org. alternatively if i understand how this works correctly, calling that could always pull  latest. i'm going to login to hub ops docker to see if i can set that up so we don't have to worry about updating here

Reply:
@betatim can you see any issue with my updating dockerhub to automagically rebuild the image each time the ea-python:latest is updated? also i can't see how you actually linked this image to dockerhub. is CI building this somewhere and pushing to dockerhub? because we could also let dockerhub manage the build. just curious if there is a slightly more efficient workflow.

Reply:
is it this deploy script:
https://github.com/earthlab/hub-ops/blob/master/.travis.yml#L70 that seems to be where the push is happening. any reason why we can't have dockerhub automagically handle the update to dockerhub? 

https://github.com/earthlab/hub-ops/blob/master/deploy.py#L160 

i'm sure i'm missing something but
1. if we connect that dockerfile to our hubops account and then
2. setup autobuild we may be able to skip a step AND also get confirmation when the build is done from dockerhub .i could setup the webhook. but first i want to ensure i'm not missing something important -- ie why @betatim set it up this way rather than using the docker autobuild. i could see the autobuild being infinitely slower sometimes for example. OR i could potentially setup both??

Reply:
@kcranston as a test of this workflow let's change this to :

`FROM earthlab/earth-analytics-python-env`

this should pull the latest build. that is what we want. when that is updated, it will automagically rebuild the image... but let's test.